tests/functional/spec/features/validation/sum/equal_total_in_separate_section_hub.spec.js

```diff
@@ -0,0 +1,179 @@
+import TotalTurnoverPage from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/total-turnover-block.page";
+import TotalEmployeesPage from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/total-employees-block.page";
+import CompanySectionSummary from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/company-overview-section-summary.page";
+
+import TurnoverBreakdownPage from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/turnover-breakdown-block.page";
+import EmployeesBreakdownPage from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/employees-breakdown-block.page";
+import BreakdownSectionSummary from "../../../../generated_pages/validation_sum_against_total_hub_with_dependent_section/breakdown-section-summary.page";
+
+import HubPage from "../../../../base_pages/hub.page";
+import ThankYouPage from "../../../../base_pages/thank-you.page";
+
+const companyOverviewSectionID = "company-overview-section";
+const breakdownSectionId = "breakdown-section";
+
+const answerAndSubmitTurnoverBreakdownQuestion = (breakdown1, breakdown2, breakdown3) => {
+  $(TurnoverBreakdownPage.turnoverBreakdown1()).setValue(breakdown1);
+  $(TurnoverBreakdownPage.turnoverBreakdown2()).setValue(breakdown2);
+  $(TurnoverBreakdownPage.turnoverBreakdown3()).setValue(breakdown3);
+  $(TurnoverBreakdownPage.submit()).click();
+};
+
+const answerAndSubmitEmployeeBreakdownQuestion = (breakdown1, breakdown2) => {
+  $(EmployeesBreakdownPage.employeesBreakdown1()).setValue(breakdown1);
+  $(EmployeesBreakdownPage.employeesBreakdown2()).setValue(breakdown2);
+  $(EmployeesBreakdownPage.submit()).click();
+};
+
+const answerAndSubmitTotalTurnoverQuestion = (total) => {
+  $(TotalTurnoverPage.totalTurnover()).setValue(total);
+  $(TotalTurnoverPage.submit()).click();
+};
+
+const answerAndSubmitTotalEmployeesQuestion = (total) => {
+  $(TotalEmployeesPage.totalEmployees()).setValue(total);
+  $(TotalEmployeesPage.submit()).click();
+};
+
+describe("Feature: Validation - Sum of grouped answers to equal total (Total in separate section)", () => {
+  describe("Given I start a grouped answer validation with dependent sections and complete the total turnover and total employees questions", () => {
+    beforeEach(() => {
```

Comment:
Do these need to beforeEach, I wonder if you changed the order you only need to open the questionnaire once